### ISSUE TITLE
Focus 40 5th gen not detected under windows 11 22h2

### ISSUE THREAD

**Issue by mdyer1** [nvaccess/nvda#15361]
### Steps to reproduce:

1.  Start NVDA 2023.2 RC orr 2023.1 depending on which is installed.  
2. 2.  Insure that The focus 40 5th is turned on.
3. 3.  Start NVDA.
4. 
### Actual behavior:

Braille display is not detected.
### Expected behavior:

Braille should be detected by NVDA when started.  I just tested this with my bs6 and got the same resault.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2023.2 RC.  Have also tried this with 2023.1 stable with the same resault.
#### Windows version:
windows 11 222h2 20621.2134
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2023.1 same problem.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Have not tried it.

**Comment by seanbudd** [comment]
Have you followed the user guide and installed the required drivers for these devices?

- https://www.nvaccess.org/files/nvda/documentation/userGuide.html#Hims
- https://www.nvaccess.org/files/nvda/documentation/userGuide.html#FreedomScientificFocus

Note that as per the user guide, the Focus doesn't support automatic detection, but the BS6 should assuming the correct drivers are installed
https://www.nvaccess.org/files/nvda/documentation/userGuide.html#AutomaticDetection

**Comment by seanbudd** [comment]
Actually I realize the user guide is unclear for the Focus device, it's not listed in the automatic detection section, but does say it's supported in the section for the focus device

**Comment by mdyer1** [comment]
Hi,

I installed the f9ocus drivers but still no luck at least with bluetooth.  If that matters.  I can try the hims driver If I can find it somewhere.

Matthew

BTW, the sense device does pair with n9o problem.  The focus is not even seen as a bt devices by windows which may be another matter.

**Comment by mdyer1** [comment]
Yes as a side note, my focus just doesn’t even show up as a device in windows bluetooth settings which might be a problem.  I don’t have a USB cable at the moment so can’t test it at the moment.

**Comment by zstanecic** [comment]
Hi Matthew,

In the last days windows has a problem with Bluetooth braille displays. Maybe Microsoft is aware of the issue.

**Comment by seanbudd** [comment]
>Yes as a side note, my focus just doesn’t even show up as a device in windows bluetooth settings which might be a problem.  I don’t have a USB cable at the moment so can’t test it at the moment.

Given this, this is definitely a driver issue not an NVDA issue

**Comment by Adriani90** [comment]
Some things to check also:
* Did you enable braille display in narator? In some cases the braille display will be totally overtaken by Narator and NVDA will not be able to recognize it anymore.
* In the user config folder in %appdata% / NVDA / Drivers, try to delete the driver for the focus braille display, actually for this one a native driver is already included as per #8853. Note only if you want to connect it via USB you need the USB driver from Freedom Scientific.
* Note also for Hims braille displays, a native driver is integrated into NVDA, you don't need an external driver.

cc: @Qchristensen I think some user guide adjustments are needed here.

**Comment by jcsteh** [comment]
I ran into this problem as well. In case you're seeing the same problem:

1. Press Windows+i to open Windows Settings.
2. Search for bluetooth and choose "Bluetooth and other devices settings".
3. Tab (quite a few times) until you get to a combo box called "Bluetooth devices discovery".
4. Change this to "Advanced".

If you don't do this, the Focus will just never show up as a possible device when trying to pair it. Ridiculous default Windows behaviour, but there it is.

**Comment by jcsteh** [comment]
Btw, you do not need the Freedom Scientific drivers installed to connect via Bluetooth. You do need them installed if you want to connect via USB.
